pkg/network/usm/usm_http2_monitor_test.go

```diff
@@ -767,6 +770,14 @@ func (s *usmHTTP2Suite) TestRawTraffic() {
 				return [][]byte{framer.bytes()}
 			},
 			expectedEndpoints: map[usmhttp.Key]int{
+				{
+					Path:   usmhttp.Path{Content: usmhttp.Interner.GetString(http2DefaultTestPath)},
+					Method: usmhttp.MethodGet,
+				}: 1,
+				{
+					Path:   usmhttp.Path{Content: usmhttp.Interner.GetString(http2DefaultTestPath)},
+					Method: usmhttp.MethodPost,
+				}: 1,
```

Comment:
that's great, but where do you test the scenario those methods are being sent as literal?
also, you should add tests for encoded versions

Reply:
added tests for both scenarios